rnd/autogpt_server/graph_templates/Discord Chatbot with History_v145.json

```diff
@@ -0,0 +1,384 @@
+{
+  "id": "5f72e49b-ea7f-4877-8889-ecc8e92b013f",
+  "version": 145,
+  "is_active": true,
+  "is_template": false,
```

Comment:
to make it show up as a template we need to change 
``"is_active": true,`` to be ``"is_active": false,``
as well as
``"is_template": false,`` should be ``"is_template": true,``